### ISSUE TITLE
docker networking with default gateway issue

### ISSUE THREAD

**Issue by mahald** [linka-cloud/d2vm#60]
the command is not working if a network with a default gateway is used. for my build of other containers i fixed this by adding "--network host" on the build command. Is there any way to tell d2vm to add "--network host" to the build command ?

the issue i get is "W: Failed to fetch http://deb.debian.org/debian/dists/bookworm/InRelease  Connection timed out [IP: 151.101.2.132 80]" im using the container to build it. it's wokring on my local system but not on my cloud build server, since the build server has only an internal network and adds a route default route/gateway for internet downloads.

basically i should be able to solve it by adding "--network host" to the build command, is there any way give additional parameters for the used build command ?

**Comment by Adphi** [comment]
Hello @mahald !

No, I'm sorry.
